web/style.css

```diff
@@ -8,4 +8,44 @@ html,
 body {
     width: 100%;
     height: 100%;
-}
\ No newline at end of file
+}
+
+.center {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+#loading-screen {
+    background-color: #282828;
```

Comment:
This color should match the in-game splash screen color, right?

I'd put that in a nice css variable at the top so it's extra clear that this needs to be touched by consumers of the template.

```css
:root {
    --splash-screen-bg-color: #282828;
}

/* ... */

#loading-screen {
    background-color: var(--splash-screen-bg-color);
}
```

Reply:
I hadn't considered it, but it's probably good if it matches, yeah.